src/components/mainPage/MainPage.css

```diff
@@ -0,0 +1,14 @@
+
+@media only screen and (min-width: 750px  ) {
+
+    .main_page_container {
+        margin: 3%;
+        width: 163vh;
```

Comment:
This doesn't make any sense. Width relative to viewport height? What are you planning to achieve?